charts/gardener-extension-acl/templates/deployment.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
   name: {{ include "name" . }}
   namespace: {{ .Release.Namespace }}
   labels:
-    high-availability-config.resources.gardener.cloud/type: webhook
```

Comment:
This should be set to `controller`, I think, see https://github.com/gardener/gardener/blob/master/docs/development/high-availability-of-components.md#convenient-application-of-these-rules

Reply:
You are right :)